extensions/skyportal/skyportal/handlers/api/archive.py

```diff
@@ -0,0 +1,163 @@
+from penquins import Kowalski
+
+from baselayer.log import make_log
+from baselayer.app.access import auth_or_token
+from baselayer.app.env import load_env
+from ..base import BaseHandler
+
+
+env, cfg = load_env()
+log = make_log("archive")
+
+
+gloria = Kowalski(
+    token=cfg["app.gloria.token"],
+    protocol=cfg["app.gloria.protocol"],
+    host=cfg["app.gloria.host"],
+    port=int(cfg["app.gloria.port"]),
+    timeout=10,
+)
+log(f"Gloria connection OK: {gloria.ping()}")
```

Comment:
Thoughts on naming the configuration section here something more generic? Like `kowaslki_archives` or something like that?

Reply:
I think the best will be to somehow implement https://github.com/dmitryduev/kowalski/issues/60 so that a separate entry in the config is unnecessary. I'd leave it as is for now and add comments with explanations to the code.
A side note that it's a pretty internal thing at this point: matchfiles are not public; light curves _are_ made public, but we don't have a reader/ingester for that (just filed https://github.com/dmitryduev/kowalski/issues/107).